home-assistant-plugin/custom_components/xknx/manifest.json

```diff
@@ -2,6 +2,6 @@
   "domain": "xknx",
   "name": "KNX",
   "documentation": "https://www.home-assistant.io/integrations/knx",
-  "requirements": [],
-  "codeowners": ["@Julius2342"]
+  "requirements": ["xknx==0.12.0"],
+  "codeowners": ["@Julius2342", "@farmio", "@marvin-w"]
```

Comment:
didn't you want to use XKNX/xknx-devs ? Doesn't it work?

Reply:
We can do that in the next PR to HA.